traffic_ops/traffic_ops_golang/ats/atsserver/parentdotconfig.go

```diff
@@ -1,4 +1,4 @@
-package ats
+package atsserver
```

Comment:
Apache Traffic Server... server?

Reply:
Apache Traffic Server, server configs. As opposed to Apache Traffic Server, profile configs.

Reply:
ah